tests/test_parse_response.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html)
+
+from cubiscan.cubiscan import CubiScan
+
+
+def test_parse_positive_measure_response():
+    scanner_obj = CubiScan('127.0.0.1', '1234')
```

Comment:
scanner obj is always the same, you could define it globally or define a unit test case